start.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# Wait for Nextcloud to come online
+if [ -n "$NC_DOMAIN" ] && [ -n "$NC_PORT" ]; then
+    while ! nc -z "$NC_DOMAIN" "$NC_PORT"; do
+        echo "Waiting for Nextcloud to start..."
+        sleep 5
+    done
```

Comment:
```suggestion
    done
    echo "Nextcloud found on $NC_DOMAIN:$NC_PORT!"
```

Reply:
Let's add some messages that would help in debugging if any problems occur.

Reply:
done